styles/stylesheet.css

```diff
@@ -0,0 +1,251 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: white;
+	height: 100%;
+	padding: 1rem;
+}
+
+.header {
+	background-repeat: no-repeat;
+	max-width: 100%;
+	height: 400px;
+	background-image: url("../images/header-image.jpg");
+	background-size: cover;
+}
+
+.header-logo {
+	max-width: 100%;
+	height: auto;
+	float: left;
+	padding: 3rem;
+}
+
+.header-logo img {
+	float: left;
+}
+
+.slogan {
+	font-family: "Comic Sans MS", cursive, sans-serif;
+	font-size: 1.5rem;
+	color: #1B8F1B
+}
+
+.shoppingcart {
+	float: right;
+	padding: 0.5rem;
+}
+
+.shoppingcart img {
+	max-width: 100%;
+	height: auto;
+}
+
+.shoppingcart p {
+	font-family: 'Arvo', serif;
+	font-size: 2.5rem;
+	color: white;
+}
+
+.checkout-button {
+	background-color: #4CAF50;
+	border: none;
+	color: white;
+	font-family: 'Arvo', serif;
+	padding: 15px 32px;
+	font-size: 1.5rem;
+	cursor: pointer;
+}
+
+nav {
+	text-align: center
+}
+
+nav ul {
+	margin: 0;
+	padding: 2rem;
+	background-color: #dddddd;
+}
+
+nav li,
+nav li a {
+	display: inline;
+	text-decoration: none;
+	font-size: 2.5rem;
+	color: #116694;
+	padding-left: 0.5rem;
+	font-weight: bold;
+}
+
+nav li a:hover,
+nav li a:active {
+	color: yellow;
+}
+
+.flex-container {
+	display: flex;
+	flex-direction: column;
+	align-items: stretch;
+}
+
+.flex-container::after {
+	content: "";
+	clear: both;
+	display: table;
+}
+
+.box {
+	flex: 2;
+	background-color: white;
+	padding-left: 3rem;
+	
+}
+
+.box img {
+	float: left;
+	padding-top: 3rem;
+	max-width: 100%;
+	height: auto;
+}
+
+.container-about {
+	font-family: 'Arvo', serif;
+	font-size: 2rem;
+}
+
+.container-about h3 {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	font-size: 3rem;
+	font-weight: bold;
+	position: relative;
+	left: 3rem;
+}
+
+.container-about p {
+	color: black;
+	font-family: 'Arvo', serif;
+	font-size: 1.2rem;
+	position: relative;
+	left: 3rem;
+}
+
+.grid-container {
+	display: grid;
+	grid-template-columns: auto auto auto auto;
+	grid-column-gap: 2rem;
+	padding: 10px;
+	max-width: 1200px;
+	margin: 0 auto;
+	grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+
+@media (min-width: 600px) {
+	.grid-container {
+		grid-template-columns: repeat(3, 1fr);
+	}
+}
+
+@media (min-width: 900px) {
+	.grid-container {
+		grid-template-columns: repeat(4, 1fr);
+	}
+}
+
+.grid-item {
```

Comment:
Your dogs don't seem to have a border around them. Make sure to double check your work with the provided spec.